pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -176,20 +178,23 @@ func (state PipelineRunState) IsBeforeFirstTaskRun() bool {
 
 // IsStopping returns true if the PipelineRun won't be scheduling any new Task because
 // at least one task already failed or was cancelled
```

Comment:
now that this function takes a dag.Graph i think updating the docstring to mention the new param would make sense - and that could explain that IsStopping only checks the statuses of tasks referenced in the passed in dag maybe?

Reply:
Done